compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -855,6 +858,24 @@ internal class RootNodeOwner(
             snapshotInvalidationTracker.requestDraw()
         }
 
+        private val isArrEnabled: Boolean = @OptIn(ExperimentalComposeUiApi::class) ComposeUiFlags.isAdaptiveRefreshRateEnabled
```

Comment:
Do we really need to store such boolean here? It's only valid that we want to remember it despite of changes by user.
My suggestion here is to just unwrap/inline reading `ComposeUiFlags` directly